tokenLists/junocw20.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "protocol": "Juno Network",
+    "symbol": "LUNO (Juno)",
+    "token": "juno13c57ssxvlzefsj4v5spdz4m9r6c6s2far5npvmc9en7nz02xqjyqne40gk",
+    "icon": "/img/luno.png",
+    "decimals": 6,
+    "originDenom": "cw20:juno13c57ssxvlzefsj4v5spdz4m9r6c6s2far5npvmc9en7nz02xqjyqne40gk",
+    "originChainId": "juno-1"
+  }
+]
```

Comment:
Astroport isn't deployed in JUNO (yet), so do not create this file. Please delete.